docker-compose.yml

```diff
@@ -28,6 +28,7 @@ services:
       BIGCHAINDB_DATABASE_HOST: mdb
       BIGCHAINDB_DATABASE_PORT: 27017
       BIGCHAINDB_SERVER_BIND: 0.0.0.0:9984
+      BIGCHAINDB_WSSERVER_HOST: 0.0.0.0
     ports:
       - "9984"
```

Comment:
I have this change here which does similar things. If you can merge it here, I won't create another PR.